app/controllers/mission_control/jobs/workers_controller.rb

```diff
@@ -2,11 +2,12 @@ class MissionControl::Jobs::WorkersController < MissionControl::Jobs::Applicatio
   before_action :ensure_exposed_workers
 
   def index
-    @workers = MissionControl::Jobs::Current.server.workers.sort_by { |worker| -worker.jobs.count }
+    @workers_page = MissionControl::Items::Page.new(workers, page: params[:page].to_i)
```

Comment:
I understand the idea of changing this name, but the `MissionControl::Jobs` namespace is because of the gem, which is called `mission_control-jobs`. It's the namespace under which the engine is isolated: https://github.com/basecamp/mission_control-jobs/blob/9e07a5d262cfc837624ff84f9944a41fa61c1bb0/lib/mission_control/jobs/engine.rb#L11
We want to keep everything nested under `MissionControl::Jobs`. In this case, we want to keep this being `MissionControl::Jobs::Page`, for example, no need to add `Items` since we just have this `Page` class.